### PR TITLE
Fix load tests

### PR DESCRIPTION
**Pull request by jingerbread**
Fix load tests on BareMetal clusters

### CODE REVIEW

File: pytest_tests/resources/load_ssh_key [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```
Comment (abereziny) [inline pytest_tests/resources/load_ssh_key:1]: It's not good to store private keys in public repo (or any repo) in my opinion.

File: pytest_tests/testsuites/load/test_load.py [hunk 2]
```diff
@@ -22,53 +23,61 @@
...
 
 
 @pytest.mark.load
 class TestLoad(ClusterTestBase):
-    @pytest.fixture(autouse=True)
-    def clear_cache_and_data(self, hosting: Hosting):
-        clear_cache_and_data(hosting=hosting)
-        yield
-        start_stopped_nodes()
+    # @pytest.fixture(autouse=True)
```
Comment (abereziny) [inline pytest_tests/testsuites/load/test_load.py:51]: How long load will work until we exaust all resources? 
In general dev environment we don't have much. 
If we don't this code then we must delete it :)

File: pytest_tests/testsuites/conftest.py [hunk 3]
```diff
@@ -30,7 +30,7 @@
     BACKGROUND_WRITERS_COUNT,
     LOAD_NODE_SSH_PRIVATE_KEY_PATH,
     LOAD_NODE_SSH_USER,
-    LOAD_NODES,
+    LOAD_NODE,
```
Comment (abereziny) [inline pytest_tests/testsuites/conftest.py:33]: One load node? 
Please revert.

File: pytest_tests/resources/load_params.py [hunk 4]
```diff
@@ -15,13 +15,15 @@
...
 READERS = os.getenv("READER", "0").split(",")
 DELETERS = os.getenv("DELETERS", "0").split(",")
+# only for s3 tests
+REGION = os.getenv("REGION", "load-1-4").split(",")
 LOAD_TIME = os.getenv("LOAD_TIME", "200").split(",")
-LOAD_TYPE = os.getenv("LOAD_TYPE", "grpc").split(",")
-LOAD_NODES_COUNT = os.getenv("LOAD_NODES_COUNT", "1").split(",")
-STORAGE_NODE_COUNT = os.getenv("STORAGE_NODE_COUNT", "4").split(",")
+LOAD_TYPE = os.getenv("LOAD_TYPE", "s3").split(",")
+LOAD_NODE = os.getenv("LOAD_NODE", "10.78.69.101").split(",")
```
Comment (abereziny) [inline pytest_tests/resources/load_params.py:25]: Private ip in public repo.

File: pytest_tests/resources/load_params.py [hunk 5]
```diff
@@ -1,9 +1,9 @@
 import os
 
 # Load node parameters
-LOAD_NODES = os.getenv("LOAD_NODES", "").split(",")
-LOAD_NODE_SSH_USER = os.getenv("LOAD_NODE_SSH_USER", "root")
-LOAD_NODE_SSH_PRIVATE_KEY_PATH = os.getenv("LOAD_NODE_SSH_PRIVATE_KEY_PATH")
+# LOAD_NODES = os.getenv("LOAD_NODES", "").split(",")
+LOAD_NODE_SSH_USER = os.getenv("LOAD_NODE_SSH_USER", "service")
+LOAD_NODE_SSH_PRIVATE_KEY_PATH = os.getenv("LOAD_NODE_SSH_PRIVATE_KEY_PATH", "./pytest_tests/resources/load_ssh_key")
```
Comment (abereziny) [inline pytest_tests/resources/load_params.py:6]: The key will work without specifying if it placed to correct path and will be picked up by paramiko automatically.

**Comment by roman-khimov** [comment]
Obsolete

**Pull request closed without merging**